src/r5py/r5/custom_cost_transport_network.py

```diff
@@ -0,0 +1,399 @@
+#!/usr/bin/env python3
+
+"""Subclass for TransportNetwork, enables custom cost routing."""
+import com.conveyal.r5
+from r5py.r5.transport_network import TransportNetwork
+from r5py.util.custom_cost_conversions import (
+    convert_java_hashmap_to_python_dict,
+    convert_python_custom_costs_to_java_custom_costs,
+)
+from r5py.util.exceptions import CustomCostDataError
+from r5py.util.jvm import start_jvm
+
+__all__ = ["CustomCostTransportNetwork"]
+
+start_jvm()
+
+
+def r5_supports_custom_costs():
```

Comment:
Could this function go into the `r5.utils` somewhere? (I saw you use it outside of this file, as well, and it would be nice to not have a global function in `__all__`, here)

Reply:
Or, alternatively, add this as a class method to `CustomCostTransportNetwork`

Reply:
good idea, moved to r5py.utils.classpath, hopefully that is suitable place for it, the file has "find_r5_classpath" so sounds that it handles similar logic